indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/models/Other.kt

```diff
@@ -64,3 +64,57 @@ data class TailsConfig(val baseDir: String, val uriPattern: String = "")
  * Abstracts blob storage reader and writer which are used for tails file management
```

Comment:
I really don`t like name of this file. May be General, common, etc.?

Reply:
Yes, agree.